src/components/SplitButton/SplitButtonMenu/__tests__/splitButtonMenu-tests.jest.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+import SplitButtonMenu from "../SplitButtonMenu";
+import MenuItem from "../../../Menu/MenuItem/MenuItem";
+import { ComponentDefaultTestId, getTestId } from "../../../../tests/test-ids-utils";
+
+const id = "menu";
+const menuDataTestId = getTestId(ComponentDefaultTestId.MENU, id);
+
+const renderSplitButtonMenu = () => {
+  return render(
+    <SplitButtonMenu id={id}>
+      <MenuItem title="Test 1" />
+      <MenuItem title="Test 2" />
+    </SplitButtonMenu>
+  );
+};
+
+describe("SplitButtonMenu", () => {
+  it("renders correctly", () => {
+    const { container } = renderSplitButtonMenu();
+    expect(container).toBeInTheDocument();
+  });
+
+  it("should pass the correct props to the underlying Menu component", () => {
```

Comment:
```suggestion
  it("should set first item as aria-activedescendant", () => {
```

is more accurate, no?